executor/executor_xapi.go

```diff
@@ -170,6 +170,7 @@ type lookupTableTask struct {
 	rows    []*Row
 	cursor  int
 	done    bool
+	respCh  chan error
```

Comment:
The channel doesn't have to return error, name it errChan is a little confusing.